Add features section (#22)
inveniosoftware/docs-invenio-ils: docs/features/interlibrary_loans.md

# Interlibrary Loans

> Invenio ILS facilitates interlibrary loans, enabling resource sharing between libraries and expanding the availability of materials to users.

Reviewer: ```suggestion
Invenio ILS facilitates management of interlibrary loans on the recipient library side, expanding the availability of materials to users.
```

## Interlibrary Collaboration

> The system supports borrowing and lending between libraries, helping to maximize resource utilization and provide users with access to a wider array of materials. This feature promotes collaboration and resource sharing among libraries.

Reviewer: isn't it the duplication of the section above?
